docs/checkers/index.md

```diff
@@ -2098,6 +2108,13 @@ def example1(lst: List[int]) -> int:
     return s
 ```
 
+Comprehension:
+
+```python
+def example7(lst: List[int]) -> List[int]:
+    return [lst[i] for i in lst]  # Fixed
```

Comment:
This correction doesn't really make sense. The variable `i` isn't a good variable name, as `i` doesn't refer to an index, but an element of the list.